spacy/training/converters/conllu_to_docs.py

```diff
@@ -246,7 +251,8 @@ def conllu_sentence_to_doc(
         deps=deps,
         heads=heads,
     )
-    doc_x.ents = [Span(doc_x, ent.start, ent.end, label=ent.label) for ent in doc.ents]
+    if ents is not None:
```

Comment:
I find it slightly confusing why this checks the `ents` var - semantically it would be the same as checking `set_ents`, no? I feel like that would make the code easier to read, but maybe I'm missing something.

Reply:
It is this way because it was a per-sentence check in the first draft, and it was the most minimal way to know the results of the check in the second spot without an additional variable. But there are some corpora where `O` is missing, so a top-level check is better. (All the code was there to do this check, so I'm not sure when it got omitted in the past.)

I don't really care either way...

Reply:
I think it's just a tad more explicit the other way around :-)